### PR TITLE
fix: export MenuOption and MenuOptions types

### PR DESCRIPTION
**Pull request by Xstoudi**

### CODE REVIEW

**Comment by stropitek** [comment]
Do you need it for the types or for the `MenuItems` component? `MenuItems` seems to be an internal component used by `DropdownMenu`, or am I wrong?

**Comment by Xstoudi** [comment]
I want to type the variable I pass to `DropdownMenu`'s `options`.

I also can export `MenuOption` and `MenuOptions` without exporting whole `MenuItems`.

**Comment by stropitek** [comment]
> I also can export `MenuOption` and `MenuOptions` without exporting whole `MenuItems`

Yes I think that would be better

File: src/components/dropdown-menu/index.ts [hunk 1]
```diff
@@ -1 +1,2 @@
 export * from './DropdownMenu';
+export { MenuOptions, MenuOption, MenuDivider } from './MenuItems';
```
Comment (targos) [inline src/components/dropdown-menu/index.ts:2]:
```suggestion
export type { MenuOptions, MenuOption, MenuDivider } from './MenuItems';
```

**Review by stropitek — APPROVED** [review]

**Pull request merged**